### DISCUSSION TITLE
Fast-csg warning that doesn't happen using standard version.  Is this really an issue?

### DISCUSSION THREAD

**Discussion by rhandel** [kennetek/gridfinity-rebuilt-openscad#7] [Q&A] [answered]
Is this really an issue?  I get this warning using fast-csg only with the development version of OpenSCAD, (2022.09.16.ci12154).  Current release 2021-01 does not produce this error.

WARNING: [fast-csg] Corefinement corefinement mesh difference #27 failed with an error: CGAL ERROR: assertion violation!
Expr: it_poly_hedge!=edge_to_hedge.end()
File: /mxe/usr/x86_64-w64-mingw32.static.posix/include/CGAL/Polygon_mesh_processing/internal/Corefinement/Visitor.h
Line: 1278

Please enlighten me.

Thanks,

RAH

**Comment by kennetek** [comment]
I honestly don't know. Whenever I get the mesh warnings, I assume its because of shared faces or edges between subtracted objects. Usually means the program is trying to make a zero-thickness feature, which is not fun. OpenSCAD seems to handle these cases though, hence why it is just a warning, so I wouldn't worry too much about it.

**Comment by rhandel** [comment, marked as answer]
It also doesn't happen on the current production release.  So I wasn't too worried.  And as aside, when I sliced the model it didn't require MeshMixer to repair the STL.  So a non-issue.  I am just doing a deeper dive of the features of SCAD.

My prints have been flawless.

RAH
